src/index.ts

```diff
@@ -20,12 +30,74 @@ import makeFragment from './utils/makeFragment';
  * @property {string} placeholder - placeholder for the empty paragraph
  * @property {boolean} preserveBlank - Whether or not to keep blank paragraphs when saving editor data
  */
+interface ParagraphConfig extends ToolConfig {
```

Comment:
let's export it. It can be useful. 
```suggestion
export interface ParagraphConfig extends ToolConfig {
```